stylesheets/portfolio.css

```diff
@@ -0,0 +1,39 @@
+.content {
+    display: flex;
+    flex-flow: row wrap;
+    column-gap: 2vw;
+    row-gap: 4vh;
+    justify-content: space-evenly;
+    align-items: stretch;
+    padding-top: 2vh;
+    padding-bottom: 2vh;
+}
+
+.projects {
```

Comment:
on your portfolio page I was playing around with adding a max height to your projects class. Right now the footer doesn't go under your projects. You can fix this by adding a max height with a vh value. This does mess up the text a little on the projects, so you would have to problem fix the size of the text or use wrap text.